WebRoot/js/zimbraAdmin/common/ZaItem.js

```diff
@@ -115,6 +115,32 @@ function(a,b) {
 	return ZaItem.compareAttr(a,b,"description");
 }
 
+ZaItem.compareEmailAddressAsc = 
+function(a,b) {
+	var al = a.emailAddress.toLowerCase();
+	var bl = b.emailAddress.toLowerCase();
+
+	if (al < bl)
+		return -1;
+	if (al > bl)
+		return 1;
+	else
+		return 0;
+}
+
+ZaItem.compareEmailAddressDesc = 
```

Comment:
can we club this function with above function? lets have 3rd parameter to identify asc or desc.

Reply:
ok